apps/api/src/server/applications/application/project-updates/project-updates.service.js

```diff
@@ -53,13 +54,18 @@ export async function updateProjectUpdateService(body, projectUpdateId) {
 	const update = await updateProjectUpdate(projectUpdateId, updateReq);
 
 	if (update.case && update.case.reference) {
-		await eventClient.sendEvents(
-			NSIP_PROJECT_UPDATE,
-			[buildProjectUpdatePayload(update, update.case.reference)],
-			EventType.Update
-		);
+		const events = [buildProjectUpdatePayload(update, update.case.reference)];
+		await eventClient.sendEvents(NSIP_PROJECT_UPDATE, events, EventType.Update);
```

Comment:
I think we only need to send one event (since ODW will treat publish/unpublish. the same as update). So might make sense to figure out what the event type is and then just send it once, like:

```
const eventType =
  update.status === ProjectUpdate.Status.published
    ? EventType.Publish
    : update.status === ProjectUpdate.Status.unpublished
    ? EventType.Unpublish
    : EventType.Update;

const events = [buildProjectUpdatePayload(update, update.case.reference)];
await eventClient.sendEvents(NSIP_PROJECT_UPDATE, events, EventType.Update);
```

But that's also a nasty ternary to read!

Reply:
OK that makes sense, and I was the question I was trying to form yesterday - sometimes easier to just see in code!

I will refactor, but not with a double ternary 😅

Reply:
How is this? https://github.com/Planning-Inspectorate/back-office/pull/1191/commits/84c1e507d65cfccf58d18582aea5c21096d7b562